perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -238,7 +313,7 @@ def WriteObjects(storage_schema, bucket, object_prefix, count,
       objects_written.append(object_name)
     except:
       logging.exception('Caught exception while writing object %s.',
-                        object_path)
+                        object_name)
```

Comment:
The purpose here is to be fault tolerant on any failed deletes and move forward with the next delete. So catching all exception is proper at here....